docs/source/getting-started/authenticators-users-basics.md

```diff
@@ -3,12 +3,12 @@
 # Authentication and User Basics
 
 The default Authenticator uses [PAM][] (Pluggable Authentication Module) to authenticate system users with
-their username and password. With the default Authenticator, any user
+their usernames and passwords. With the default Authenticator, any user
 with an account and password on the system will be allowed to login.
 
 ## Create a set of allowed users (`allowed_users`)
 
-You can restrict which users are allowed to login with a set,
+You can restrict which users are allowed to login with a set
```

Comment:
Can we get this comma back? It was correct before.

Reply:
Alright, I will do that right away.